examples/QueryDsl/MultiMatchQueryPage/047266b0d20fdb62ebc72d51952c8f6d.asciidoc

```diff
@@ -0,0 +1,21 @@
+////
+IMPORTANT NOTE
+==============
+This file is generated from method Line341 in https://github.com/elastic/elasticsearch-net/tree/master/src/Examples/Examples/QueryDsl/MultiMatchQueryPage.cs#L345-L377.
+If you wish to submit a PR to change this example, please change the source method above
+and run dotnet run -- asciidoc in the ExamplesGenerator project directory.
+////
+[source, csharp]
+----
+var searchResponse = client.Search<object>(s => s
+    .Index(Nest.Indices.AllIndices)
+    .Query(q =>
+        q.MultiMatch(c => c
+            .Query("Will Smith")
+            .Type(TextQueryType.CrossFields)
+            .Fields("first_name, last_name")
```

Comment:
```suggestion
            .Fields(new[] { "first_name", "last_name" })
```

Reply:
should we _discourage_ the comma separated string approach?

Reply:
It think its a matter of taste, but will move to arrays to be consistent.